src/aiida_wannier90_workflows/workflows/optimize.py

```diff
@@ -76,7 +88,7 @@ class Wannier90OptimizeWorkChain(Wannier90BandsWorkChain):
     # chk file by setting `restart = plot` in wannier90.win.
     _WANNIER90_PLOT_INPUTS = (
         "wannier_plot",
-        "bands_plot",
+        # "bands_plot",
```

Comment:
Would be great if you could add some explanations before the commented `bands_plot`, sth like
```
the bands_plot is commented out since it is rather cheap to compute, and also we want to check the band distance during each iteration of optimizing dis_proj_min/max even if we set `separate_plotting = True`
```